src/datapump/jobs/version_update.py

```diff
@@ -137,14 +137,6 @@ def _create_aux_tile_set(self, tile_set_parameters: RasterTileSetParameters) ->
                 "num_processes": co.num_processes,
                 "resampling": co.resampling,
             },
-            "metadata": {
```

Comment:
why was this removed?

Reply:
This was a hotfix for the new metadata schema that I just pulled from `master`.  This was removed because it was specifying version level metadata for assets.